frontend/public/components/pod.jsx

```diff
@@ -193,6 +193,26 @@ const Details = ({obj: pod}) => {
 
     </div>
 
+    <div className="co-m-pane__body">
+      <h1 className="co-section-title">Init Containers</h1>
+      <div className="row">
```

Comment:
I'd make this a shared components that you pass an array of containers to. Then we don't have to repeat the markup twice.

cc @benjaminapetersen

Reply:
Agree, def can share & eliminate the duplication.

Reply:
@sg00dwin you can probably replace lines `196-214` and lines `216-234` with a single component, called `<ContainerRow>` or something. 

It might look something like:
`<ContainerRow heading="Containers" containers={pod.spec.containers}>`  
and 
`<ContainerRow heading="Init Containers" containers={pod.spec.initContainers}>`
and then somewhere above you do:
```
const ContainerRow = (heading, containers) => { return <div className="co-m-pane__body"> .... (all that html) }
```
There might be an even more elegant way to do it, but more or less thats the direction.

Reply:
I would call it `ContainersTable` if it's the whole table.

I'd suggest keeping the heading outside the component so we can show different headings for init vs normal containers.

Reply:
Agree, it is a table so `row` isn't great.  && Agree on leaving out the heading, makes it an even easier component to extract.